translations/zh/built-in-makers-zh.md

```diff
@@ -0,0 +1,80 @@
+内置制造机 Built In Makers
+===============
+下面是一个Fili内置标准制造机表格。对于标准制造机，请参考[Configuring Metrics][configuring metrics]。
+
+Druid集合操作制造机
+---------------------------------------
+
+下列制造机用于制造一级标准，且与下面表格中的操作一一对应。
+[Druid aggregations][druid aggregations]。
+
+
+1. [`长整数之和制造机LongSumMaker`][长整数之和制造机LongSumMaker]
+2. [`双精度之和制造机DoubleSumMaker`][精度之和制造机DoubleSumMaker]
+3. [`长整数最大值制造机LongMaxMaker`][长整数最大值制造机LongMaxMaker]
+4. [`双精度最大值制造机DoubleMaxMaker`][双精度最大值制造机doubleMaxMaker]
+5. [`长整数最小值制造机LongMinMaker`][长整数最小值制造机LongMinMaker]
+6. [`双精度最小值制造机DoubleMinMaker`][双精度最小值制造机DoubleMinMaker]
+7. [`计数器制造机CountMaker`][计数器制造机CountMaker]<sup>[1](#countCaveat)</sup>
+
+另外，Fili的核心库支持Sketch类型和集合操作。详见
+[theta sketches in Druid][druid sketch module]。
+
+8. [`Sketch计数器制造机SketchCountMaker`][Sketch计数器制造机SketchCountMaker] -
+sketchCount与[sketch aggregation][sketch module]相关联。
+
+关于sketch的更多内容可参考http://datasketches.github.io/.
+
+Druid后集合操作制造机
+--------------------------------------------
+
+下列制造机在查询上使用单次Druid后集合运算符。关于这些运算请参见
+[Druid post-aggregations][druid post-aggregations]
+
+1. [`运算制造机ArithmeticMaker`][运算制造机ArithmeticMaker]
+2. [`常量制造机ConstantMaker`][常量制造机ConstantMaker]
+
+Fili也支持Sketch集合操作。
+
+3. [`Sketch集合运算制造机SketchSetOperationMaker`][Sketch集合运算制造机SketchSetOperationMaker] - 是[Druid sketch module][sketch module]的一部分。
+
+我们现在并不支持源生Druid `JavaScript` 及 `HyperUnique Cardinality`后集合运算符。
+Fili支持sketches而不是hyperUnique对象，因为二者功能重复。`JavaScript`后运算符在大规模运行中会造成性能瓶颈。
+
+建造操作制造机
```

Comment:
We say it's constructed operation because some data aggregations are not simply done by Druid. One example is **average**. Average is a **Fili aggregator**, which is more complex than simply taking some summation. Could you come up with a better translation?

Reply:
I don't understand this. Could you please expand a little bit?